sqlalchemy_imageattach/entity.py

```diff
@@ -127,8 +127,16 @@ def image_attachment(*args, **kwargs):
        implementation.
 
     """
+    if 'has_many' in kwargs:
```

Comment:
Because `image_attachment()` is a wrapper for `relationship()`, I think there is confliction of arguments. If user put `uselist=False` to disable multiple imageset feature, it will make the result to scalar value, not the list of images.